libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -81,6 +81,7 @@ class AP_BattMonitor
     // battery failsafes must be defined in levels of severity so that vehicles wont fall backwards
     enum class Failsafe : uint8_t {
         None = 0,
+        Unhealthy,
```

Comment:
need to check if this is exposed anywhere (logging? lua scripting?) - this changes the numbering

Reply:
I have confirmed this is not exposed. I changed the enum name and the state variable name and built to make sure I checked all the users.